com.unity.visualeffectgraph/Editor/Models/VFXGraph.cs

```diff
@@ -55,6 +68,11 @@ static void OnCompileResource(VisualEffectResource resource)
         {
             if (resource != null)
             {
+                var path = AssetDatabase.GetAssetPath(resource);
+                var guid = AssetDatabase.AssetPathToGUID(path);
+                if (!s_SanitizedGraph.Contains(guid))
+                    return; //Early return, the reimport will be forced with the next OnPostprocessAllAssets after Sanitize
```

Comment:
TODO : Check behavior of OnSetupMaterial

Reply:
Fixed at https://github.com/Unity-Technologies/Graphics/pull/4971/commits/ded88dc43000aa6f7e1686faa781c39108c2f766